.ci/test/behat/cleanup

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+set -eo pipefail
+
+#
+# This script runs cleanup after Behat tests complete.
+#
+
+# Authenticate with Terminus
+terminus -n auth:login --machine-token="$TERMINUS_TOKEN"
+
+# Restore the DB backup made before testing
+terminus -n backup:restore $TERMINUS_SITE.$TERMINUS_ENV --element=database --yes
```

Comment:
I'm still -1 on the backup / restore workflow for tests. I think it's dangerous; far better to make non-destructive tests. I know you typically use this pattern on the WordPress side, but I thought that was mostly due to the fact that  you didn't have the features of the Drupal Driver to create fixtures on the fly that are automatically deleted at the end of a test.

If we do employ a backup/restore workflow, perhaps it could be disabled by default, and folks could set a well-known environment variable in their CI environment variables to turn it on.

Reply:
> I know you typically use this pattern on the WordPress side, but I thought that was mostly due to the fact that you didn't have the features of the Drupal Driver to create fixtures on the fly that are automatically deleted at the end of a test.

That is true. Feel free to push a commit to this PR that does the cleanup with the features of the Drupal Driver.